common-libs/internal-api-clients/src/helpers/headers.ts

```diff
@@ -0,0 +1,30 @@
+type createHeadersOptions = {
+  accessApiKey: string
+  sdkVersion?: string
+}
+
+const defaultHeaderValue = 'unknown'
+
+export type ApiRequestHeaders = { [key: string]: string }
+
+export const createHeaders = (options: createHeadersOptions): ApiRequestHeaders => {
+  return {
+    Accept: 'application/json',
+    'Api-Key': options.accessApiKey,
+    'Content-Type': 'application/json',
+    'X-SDK-Version': options.sdkVersion || defaultHeaderValue,
+  }
+}
+
+type updateHeadersOptions = {
+  authorization?: string
+  storageRegion?: string
+}
+
+export const updateHeaders = (headers: ApiRequestHeaders, options: updateHeadersOptions) => {
```

Comment:
The name is misleading, as if this method changes the input value.

An example of a better name would be `getExtendedHeaders`.